internal/datacoord/handler.go

```diff
@@ -331,7 +331,7 @@ func (h *ServerHandler) CheckShouldDropChannel(channel string) bool {
 			}
 		}
 		return false*/
-	return h.s.meta.catalog.IsChannelDropped(h.s.ctx, channel)
+	return !h.s.meta.catalog.IsChannelDropped(h.s.ctx, channel)
```

Comment:
Shall not revert here. IsChannelDropped is determined by an etcd flag for any transaction failure

Reply:
Yes but I have flipped the method's return values

Reply:
Then the method does not behave like expected
https://github.com/milvus-io/milvus/blob/403f85a95fbe60340ae7281ab8161347a8fb130e/internal/metastore/kv/datacoord/kv_catalog.go#L458-L465
Original code means whether a tombstone is marked for this channel. This tombstone was for batch save segment info and mark channel dropped.

Reply:
Per offline discussed, we decided to rename `IsChannelDropped` as `ShouldDropChannel` to improve readability.